Join Message Crashes Bot

When a user joins the server, the join message crashes the bot with a "slice bounds out of range" error.

Error Message Below:
```
panic: runtime error: slice bounds out of range [:1] with length 0

goroutine 40 [running]:
github.com/the-sanctuary/waddles/pkg/cmd.(*Router).Handler.func1(0xc0000d26c0, 0xc000678700)
        /home/rory/waddles/pkg/cmd/router.go:72 +0xb3b
github.com/bwmarrin/discordgo.messageCreateEventHandler.Handle(0xc000267320, 0xc0000d26c0, 0xa17360, 0xc000678700)
        /home/rory/go/pkg/mod/github.com/bwmarrin/discordgo@v0.22.0/eventhandlers.go:497 +0x4b
created by github.com/bwmarrin/discordgo.(*Session).handle
        /home/rory/go/pkg/mod/github.com/bwmarrin/discordgo@v0.22.0/event.go:171 +0x111
task: Failed to run task "run": exit status 2```

--- a/pkg/cmd/router.go
+++ b/pkg/cmd/router.go
@@ -69,8 +69,16 @@ func (r *Router) Handler() func(*discordgo.Session, *discordgo.MessageCreate) {
 			return
 		}
 
+		/*
+		 Check to see if the length of the message content is greater than 0
+		 For server messages, such as a Join or Leave message when a user joins or leaves a guild
+		*/
+		if len(message.Content) == 0 {
+			return
+		}
+
 		// Check to see if our prefix is there
-		if message.Content[:len(r.Prefix)] != r.Prefix { //TODO: something wrong here. See: https://github.com/the-sanctuary/waddles/issues/15
+		if message.Content[:len(r.Prefix)] != r.Prefix {
 			return
 		}
 

